### PR TITLE
[FIX] wrong new userInfo during user creation

### LINKED ISSUE #24050: [BUG] Wrong new userInfo after user creation
### Description:

When a new user is created from admin panel, after creation the context bar show information for rocket.cat.

### Steps to reproduce:

1. Goto Admin panel 
2. Create a new user from user section

### Expected behavior:

Information of the newly created user must be shown 

### Actual behavior:
Information of rocket.cat is shown

https://user-images.githubusercontent.com/50165440/147668170-5d2551a4-f6b4-4d96-b403-2a250f7b5b32.mp4

### Server Setup Information:

- Version of Rocket.Chat Server: 4.3.0-develop

### PR DESCRIPTION
**Pull request by Aman-Maheshwari**
- [x] I have read the Contributing Guide 
  - [x] I have signed the CLA 
  - [x] Lint and unit tests pass locally with my changes
  - [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
  - [ ] I have added necessary documentation (if applicable)
  - [ ] Any dependent changes have been merged and published in downstream modules

## Proposed changes (including videos or screenshots)
fixes #24050 

## Issue(s)

#24050 

## Steps to test or reproduce

Mention in the issue

## Before

https://user-images.githubusercontent.com/50165440/147667923-c45e756f-81fa-4a13-9d7a-d55a305ee4f3.mp4

## After

https://user-images.githubusercontent.com/50165440/147667934-43e4696a-5fc6-4f97-949e-1491bc0954fa.mp4

### CODE REVIEW

File: client/views/admin/users/UsersPage.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ function UsersPage() {
 						<VerticalBar.Close onClick={handleVerticalBarCloseButtonClick} />
 					</VerticalBar.Header>
 
-					{context === 'info' && <UserInfoWithData uid={id} onReload={reload} />}
+					{context === 'info' && id !== undefined && <UserInfoWithData uid={id} onReload={reload} />}
```
Comment (murtaza98) [inline client/views/admin/users/UsersPage.js:113]:
```suggestion
					{context === 'info' && id && <UserInfoWithData uid={id} onReload={reload} />}
```

Comment (murtaza98) [reply client/views/admin/users/UsersPage.js:113]: In Javascript, we don't we to explicity check for undefined, we can rather chain it like this

Comment (Aman-Maheshwari) [reply client/views/admin/users/UsersPage.js]: I have done the suggested changes. @murtaza98  please Review it

**Review by murtaza98 — APPROVED** [review]

File: client/views/admin/users/UsersPage.js (hunk 1 above)
Comment (murtaza98) [reply client/views/admin/users/UsersPage.js]: Awesome... Thanks!!

**Pull request merged**